contracts/mod-nft/src/contract.rs

```diff
@@ -22,10 +33,75 @@ pub fn execute(
     _info: MessageInfo,
     _msg: ExecuteMsg,
 ) -> Result<Response, ContractError> {
-    todo!();
+    Err(ContractError::Noop)
```

Comment:
```suggestion
    Err(ModError::Noop)
```
Do we want to move that to the `mod-sdk` package? Every mod will have only this one error

Reply:
Do you think we should move `ContractError` to `mod-sdk` and include only the error about `execute`?

Reply:
> Do you think we should move `ContractError` to `mod-sdk` and include only the error about `execute`?

I like the idea, but I'm not sure) We should call it something like `ModError` instead if we do so